config/policies/default.cedar

```diff
@@ -44,4 +44,21 @@ forbid (
     action == Access::Action::"Approve",
     resource
 )
-when { principal == resource.principal };
\ No newline at end of file
+when { principal == resource.principal };
+
+// Allow admins to read all resources
+permit (
+    principal,
```

Comment:
the policy here is a bit too permissive as it allows everyone to have admin read access, let's scope it down to suggest to the user they should do the same
```suggestion
    principal in Entra::Group::"ID_OF_COMMON_FATE_ADMINISTRATOR_GROUP",
```